Framework/TestHelpers/src/WorkspaceCreationHelper.cpp

```diff
@@ -400,6 +400,23 @@ MatrixWorkspace_sptr create2DDetectorScanWorkspaceWithFullInstrument(int nhist,
   return builder.buildWorkspace();
 }
 
+//================================================================================================================
+/** Create an Workspace2D with an instrument that contains detectors arranged at even latitude/longitude
+ * values. For use in testing absorption and multiple scattering corrections. The sparse instrument functionality
+ * in these algorithms uses geographical angles (lat/long) to specify the detector positions
+ * Latitude/longitude corresponds to two theta if longitude/latitude equals zero
+ */
+Workspace2D_sptr create2DWorkspaceWithGeographicalDetectors(const int nlat, const int nlong, const double anginc,
+                                                            const int nbins, const std::string &instrumentName) {
+  auto inputWorkspace = WorkspaceCreationHelper::create2DWorkspaceBinned(nlat * nlong, nbins, 0.5 /*x0*/);
```

Comment:
What does this `/*x0*/` comment refer to?

Reply:
It's the name of the parameter that receives the value 0.5 in the method Create2DWorkspaceBinned. I could never remember if this was the start x value or the bin width so I put this comment in. I can remove if it's a bit obscure!

Reply:
I've declared a constexpr variable called x0 and removed the comment. Hopefully that's a bit more clear now